### PR TITLE
[Backport stable/8.2] Not write the entire command for rejection

### PR DESCRIPTION
**Pull request by nicpuppa**
# Description
Backport of #14042 to `stable/8.2`.

relates to #12699

### CODE REVIEW

File: qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/CreateDeploymentTest.java [hunk 1]
```diff
@@ -157,4 +166,35 @@ public void shouldRejectDeployIfResourceIsTooLarge() {
...
+
+  @Test
+  public void shouldNotWriteResourcesInformationInRejectedRecords() {
+    // when
+    final var modelThatFitsJustWithinMaxMessageSize =
+        Bpmn.createExecutableProcess("PROCESS")
+            .startEvent()
+            .documentation("x".repeat((1046900)))
+            .done();
+    final var command =
```
Comment (koevskinikola) [inline qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/CreateDeploymentTest.java:178]: :wrench: @nicpuppa I guess we don't need the `final var command` in the test you added. It doesn't impact anything, but might be nice to remove it so the code-scan doesn't complain.

**Review by koevskinikola — APPROVED** [review]
LGTM :rocket:

**Comment by nicpuppa** [comment]
Thanks @koevskinikola, suggestion applied 💪 

bors merge

**Comment by nicpuppa** [comment]
bors retry

**Pull request merged**